pom.xml

```diff
@@ -194,8 +194,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.3</version>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
```

Comment:
Are you sure? It was a deliberate decision to support 1.7 when we created this client.